relay/api/tokens/resources.py

```diff
@@ -0,0 +1,102 @@
+from relay.blockchain.unw_eth_proxy import UnwEthProxy
+from relay.blockchain.token_proxy import TokenProxy
+
+from relay.api.schemas import TokenEventSchema, UserTokenEventSchema
+from flask_restful import Resource
+from flask import abort
+from webargs import fields
+from webargs.flaskparser import use_args
+from marshmallow import validate
+from typing import Union  # NOQA
```

Comment:
Please use f code, for example `# noqa: F401` to only disable a specific warning and not all

Reply:
done